.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+**Release note**:
+<!--  Steps to write your release note:
+1. Use the release-note-* labels to set the release note state (if you have access)
```

Comment:
So there are two way to set release-note related labels for mungegithub. One, if you have access to set label directly, just do it. Or if someone don't have access to do it or you want to add more details in release-note, then pr creator can add release message to pr description using this template. Prow will automatically add "release-note" label. If you add "None", "release-note-none" label will be added. Take a look at this [pr](https://github.com/istio/test-infra/pull/356), will help understand what's happening. If you don't do any of these two, "release-note-needed" will be added to block merging.